test/GPv2Settlement.test.ts

```diff
@@ -679,6 +683,40 @@ describe("GPv2Settlement", () => {
     });
   });
 
+  describe("executeInteraction", () => {
+    it("should fail when target is allowanceManager", async () => {
```

Comment:
Last missing test case might be what happens if there are some passing and some failing txs (could be combined with this one)

Reply:
This method doesn't take multiple interactions. This will be introduced in a coming PR.